modo/helpers.py

```diff
@@ -23,13 +24,79 @@ def dict_to_instance(element: Mapping[str, Any]) -> Any:
     )
 
 
+def set_part_of_relationship(
+    element: model.DataEntity
```

Comment:
**question:** do we really need to pass `element` around? we use `element.id` and `element.__class__.__name__`.

**comment:** I think `id` could be extracted / replaved by `element_path`. For the class name, we could either use the string like in `get_haspart_property`, or use an enum in both function if that make sense.

Reply:
An example of what I mean:
```py
def set_haspart_relationship(
    child_class: ElementType | UserElementType,
    child_path: str,
    parent_group: zarr.hierarchy.Group
):
    ...
def get_haspart_relationship(child_class: ElementType | UserElementType):
    ...
```

Or alternatively:
```py
def set_haspart_relationship(
    child_class: str,
    child_path: str,
    parent_group: zarr.hierarchy.Group
):
    ...
def get_haspart_relationship(child_class: str):
    ...
```